edge-hub/docker/linux/amd64/Dockerfile

```diff
@@ -12,7 +14,8 @@ RUN apk update && \
     apk add --no-cache snappy libcap	
 
 # Add an unprivileged user account for running Edge Hub	
-ARG EDGEHUBUSER_ID=1000	
+ARG EDGEHUBUSER_ID	
```

Comment:
This seems subtly different from this one - https://github.com/Azure/iotedge/pull/4998/files#diff-c53eb6c91226f5467c44ca4bea9845150867a5792a5a66c1d9b08d02d21ca46eR5
Just curious... Why is that?

Reply:
I changed the base image in the PR from 8 to 11. But base image had been changed on master so I had a merge conflict. Fixing it on 40 files was painful so I reverted, rebased and changed the base image again.